downstream/titles/release-notes/topics/aap-24.adoc

```diff
@@ -0,0 +1,111 @@
+
+* Although you can still synchronize content, the use of synclists is deprecated and will be removed in a later release. Instead, private automation hub administrators can upload manually-created requirements files from the `rh-certified` remote.
+
+* You can now configure the Controller Access Token for each resource with the `connection_secret` parameter, rather than the old `tower_auth_secret` parameter.  This change is backwards compatible, but the `tower_auth_secret` parameter is now deprecated and will be removed in a future release.
+
+* Smart inventories have been deprecated in favor of constructed inventories and will be removed in a future release.
+
+== Bug fixes
+
+The following bugs were fixed in this release of Ansible Automation Platform:
+
+* Installer now ensures that collection auto signing cannot be enabled without enabling the collection signing service.
+
+*  Fixed an issue with restoring backups when the installed automation controller version is different from the backup version.
+
+* Fixed an issue with not adding user defined galaxy-importer settings to `galaxy-importer.cfg` file.
+
+* Added missing `X-Forwarded-For` header information to nginx logs.
+
+* Removed unnecessary receptor peer name validation when IP address is used as the name.
+
+* Updated the `outdated base_packages.txt` file that is included in the bundle installer.
+
+* Fixed an issue where upgrading the Ansible Automation Platform did not update the nginx package by default.
+
+* Fixed an issue where an *awx* user was created without creating an *awx* group on execution nodes.
+
+* Fixed the assignment of package version variable to work with flat file inventories. 
+
+* Added a FQDN check for the automation hub hostname required to run the Skopeo commands.
+
+*  Fixed an issue such that the front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
+
+* Fixed the variable precedence for all component `nginx_tls_files_remote` variables.
+
+* Fixed the *setup.sh* script to escalate privileges if necessary for installing Ansible Automation Platform. 
+
+* Fixed an issue when restoring a backup to an automation hub with a different hostname.
+
+== Technology Preview
+
+Some features in this release are currently classified as Technology Preview. Technology Preview features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. Please note that Red Hat does not recommend using technology preview features for production, and Red Hat SLAs do not support technology preview functions.
+
+Following are the technology preview features: 
+
+* Ansible Automation Platform 2.4 adds the ability to install the automation controller for IBM Power (ppc64le), IBM Z (s390x), and IBM(R) LinuxONE (s390x) architectures as a Technology Preview feature.
```

Comment:
Is it necessary in a Technology Preview section, under a heading that says the following are technology preview features, to say that something is a technology preview feature?

Reply:
Agree, this is repetitive. Removed the 'as a tech. preview feature' text.